landlab/components/profiler/trickle_down_profiler.py

```diff
@@ -0,0 +1,392 @@
+# coding: utf8
+# ! /usr/env/python
+"""trickle_down_profiler.py component to create channel profiles."""
+from collections import OrderedDict
```

Comment:
Is this a new dependency, or is it part of standard libraries (or do we already depend on it)?

Reply:
OrderedDict is part of standard libraries and is already used elsewhere in landlab (additionally, if I needed a new external dependency it would show up in requirements.txt or similar files).